client/packages/host/src/useInitPlugins.ts

```diff
@@ -0,0 +1,48 @@
+import {
```

Comment:
I wanted to keep this file in common but had to be moved to host, small comment below, something about loading local plugins with dynamic import was being tricky when dynamic import is not in the package that is being built/run